cmd/vcert/cmdCloudKeystores.go

```diff
@@ -94,3 +92,22 @@ func doCommandProvisionCloudKeystore(c *cli.Context) error {
 	}
 	return nil
 }
+
+func gettingIDsFromFiles(flags *commandFlags) (*commandFlags, error) {
+	if flags.pickupIDFile != "" {
+		fmt.Printf("pickupIDFILE is not empty string")
```

Comment:
just an observation that this message refers to a variable name (internal to code) and may be confusing to reader -- perhaps just `Pickup ID file` as you have it in the message below?

OR

is this printf necessary? it looks a bit like a debug message...

Reply:
Fixed by latest changes